libgraphics/src/harfbuzztext.cpp

```diff
@@ -214,7 +214,7 @@ MCHarfbuzzSkiaFace *MCHarfbuzzGetFaceForSkiaTypeface(SkTypeface *p_typeface, uin
         t_hb_sk_face -> skia_face = t_face;
         p_typeface -> ref();
         
-		MCGCacheTableSet(s_hb_face_cache, t_key, sizeof(t_id), t_hb_sk_face, sizeof(MCHarfbuzzSkiaFace));
+        //MCGCacheTableSet(s_hb_face_cache, t_key, sizeof(t_id), &t_hb_sk_face, sizeof(MCHarfbuzzSkiaFace*));
```

Comment:
IIRC I implemented the face cache to combat very slow rendering of text on Android. I'm not sure if it's a good idea to comment it out entirely (although it may be worth checking). If it causes problems for emscriptem, can we use an ifdef?